doglover/src/components/AllBreeds.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+// import { Link } from "react-router-dom";
+import {Container, Image, Heading, Text, Select, Link} from '@chakra-ui/react';
+
```

Comment:
Hello Alex. Thanks for the effort. When rendering the component there are "some" console errors. Is there a way to resolve them ?